### DISCUSSION TITLE
how to remove blacklisted keys from the keybinding state

### DISCUSSION THREAD

**Discussion by StuffyAnt** [KadeArchive/Kade-Engine#1645] [Q&A] [answered]
so i tried to put in my everyday keybinds for rhythm games (ZX, NUMPAD2 and NUMPAD3) but it said that the numpad keys were blacklisted. i checked the code and there isn't really any reference to a blacklist appart from the blacklist for ENTER, TAB and the other obvious keys. is there a way to "fix" this?

**Comment by isakube** [comment, marked as answer]
Line 375:
```haxe
if (r.contains("NUMPAD"))
        {
            keys[curSelected] = null;
            lastKey = r;
            return;
        }
```
just remove this

**Reply by StuffyAnt** [reply]
Oh, i see. Quite simple, isn't it...
